Library/Homebrew/cask/dsl.rb

```diff
@@ -92,6 +94,25 @@ def name(*args)
       @name.concat(args.flatten)
     end
 
+    def license(*args)
```

Comment:
Would be good to have something similar in `formula.rb`; I think that's probably a priority over `cask/dsl.rb` (where many/most packages will have a proprietary license). Nice work so far!